### ISSUE TITLE
Yarn dependencies no longer automatically installing

### ISSUE THREAD

**Issue by editkid** [cypress-io/circleci-orb#242]
## The orb does not work as expected

v1.19.0

About ~13hrs ago ( around `2020-03-11T18:00:18.739Z`) our tests stopped automatically running `yarn install`: we ended up without a `node_modules` folder. Previously this was happening automatically.

The solution/workaround was to use the new custom `install-command`, but I assume that shouldn't be required?

**Orb version**
v1.18.0

**circle.yml**
```
      - cypress/run:
          executor: cypress
          yarn: true
          working_directory: cypress
```

**Our workaround**
```
      - cypress/run:
          executor: cypress
          working_directory: cypress
          install-command: yarn install --frozen-lockfile
```
Thank you. :v:

**Comment by fabiocarneiro** [comment]
Also happening here after the release of 1.18/1.19

Backward compatibility break. Please release a correction minor.

**Comment by fabiocarneiro** [comment]
I'm guessing it's because of the default value `''`: 
https://github.com/cypress-io/circleci-orb/blob/19a4fe204d42addc2931e3dc84cfbc4854d2dbc5/src/orb.yml#L131-L133

evaluating to true here

https://github.com/cypress-io/circleci-orb/blob/19a4fe204d42addc2931e3dc84cfbc4854d2dbc5/src/orb.yml#L151-L156

**Comment by bahmutov** [comment]
Ugghh, the beauty of making parameter `install-command` optional by giving it the default value of '' but it being truthy.

Work-around: please lock the version of orb you use to 1.17.1

```
orbs:
  cypress: cypress-io/cypress@1.17.1
```

While I think of the best way to solve this.

**Comment by bahmutov** [comment]
Released in v1.19.1
